python/python/bystro/ancestry/train.py

```diff
@@ -376,39 +501,125 @@ def sample(cls) -> "RFCParamChoices":
         return cls(**kwargs)
 
 
-def _make_rfc(
-    train_Xpc: np.ndarray,
-    test_Xpc: np.ndarray,
-    train_y: pd.Series,
-    test_y: pd.Series,
+@dataclasses.dataclass(frozen=True)
+class AccuracyReport:
+    """Represent model accuracy scores at population and superpopulation levels."""
+
+    train_pop_accuracy: float
+    test_pop_accuracy: float
+    train_superpop_accuracy: float
+    test_superpop_accuracy: float
+
+
+def _compute_accuracy_report(
```

Comment:
Annotation here and for the following function would be helpful

Reply:
Do you mean a docstring?  Added, but which other function?  Looks like make_rfc's already covered